README.md

```diff
@@ -43,6 +43,36 @@ python nutrition_labels/grant_tagger.py --training_data_file data/processed/trai
 
 where `vectorizer_type` can be 'count', 'tfidf' or 'bert', `model_type` can be 'naive_bayes', 'SVM' and 'log_reg', and `bert_type` (if using bert) can be 'bert' and 'scibert'.
 
```

Comment:
Do you mind adding this here for me rather than me causing problems including it in my PR and getting merge conflicts:

## Getting the references of the tech grants' publications

Running
```
python nutrition_labels/finding_tech_grant_refs.py
```
will find all the PMIDs of publications which acknowledge any grant numbers in the tech grant we identified from the ensemble model (given in 'data/processed/ensemble_results.csv'). We found that 1138 of the 1790 tech grants (64%) were acknowledged in publications. 

Then it uses the EPMC client to get all the references from each of these publications. This creates two useful outputs - `data/processed/tech_grants_references_dict_2020-09-16.json` information about each of the references found, and `data/processed/tech_grants_reference_ids_counter_2020-09-16.json` a counter of how many times each reference was found.